src/perfiles/infrastructure/mappers.py

```diff
@@ -114,3 +116,25 @@ def dto_to_entity(self, dto: PerfilAlimenticioDTO) -> PerfilAlimenticio:
             tipo_identificacion=dto.tipo_identificacion,
             identificacion=dto.identificacion,
         )
+
+class HabitoDeportivoMapper(Mapper):
```

Comment:
Para PerfilDeportivoMapper:

```suggestion
def dto_to_entity(self, dto: PerfilDeportivoDTO) -> PerfilDeportivo:
    mapper = HabitoDeportivoMapper*()
    habits = [mapper. dto_to_entity(h) for h in dto.habitos]
```